.circleci/config.yml

```diff
@@ -0,0 +1,148 @@
+version: 2.1
+
+aliases:
+  - &workspace
+    ~/chat-demo-app
+  - build-type-parameter: &build-type-parameter
+      build-type:
+        description: "Build type to build"
+        type: string
+
+executors:
+  android-executor:
+    docker:
+      - image: cibuilderbot/docker-circleci-linux-android
+    working_directory: *workspace
+    resource_class: xlarge
+    environment:
+      JVM_OPTS: -Xmx15g
+
+commands:
+  download_dependencies:
+    description: Download Dependencies
+    steps:
+      - restore_cache:
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum  "app/build.gradle" }}
+      - run:
+          name: Download Dependencies
+          command: ./gradlew androidDependencies
+      - save_cache:
+          paths:
+            - ~/.gradle
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum  "app/build.gradle" }}
+
+  generate_gcloud_settings:
+    description: "Generate GCloud settings"
+    steps:
+      - run:
+          name: Decode Google Cloud key.json
+          command: echo "$GCLOUD_SETTINGS" | base64 --decode > rtd-shared-lib-a27041a3fda9.json
+      - run:
+          name: Set Google Cloud target project
+          command: gcloud config set project rtd-shared-lib
+      - run:
+          name: Authenticate with Google Cloud
+          command: gcloud auth activate-service-account  rtd-shared-lib@appspot.gserviceaccount.com --key-file=rtd-shared-lib-a27041a3fda9.json --project=rtd-shared-lib
```

Comment:
Move acc name to env var? Generate json file name from same var.

Reply:
json file name is hardcoded, project name moved to env var